File: sukh_1684/problems/maximum_number_of_words_found_in_sentences/solution.js
/**
 * @param {string[]} sentences
 * @return {number}
 */
var mostWordsFound = (sentences)  => {
    var max = 0;
    sentences.forEach((sentence) => {
        var words = 0;
        for(var i =0; i<sentence.length;i++){
            if(sentence.charAt(i) == ' '){
                words ++;
            }
        }
        ++ words;
        if(words > max){
            max = words;
        }
            
    })
    return max;
};

File: sukh_1684/problems/maximum_number_of_words_found_in_sentences/solution.py
class Solution(object):
    def mostWordsFound(self, sentences):
        """
        :type sentences: List[str]
        :rtype: int
        """
        
        maximum = 0
        for sentence in sentences:
            wordCount = 1
            for character in sentence:
                if (character == ' '):
                    wordCount +=1
            maximum = max(maximum, wordCount)
            
        return maximum